twyla/service/queues.py

```diff
@@ -118,38 +93,25 @@ def __init__(self):
             await self.protocol.close()
 
 
-    async def emit(self, event_name, payload):
-        # Declare the queue to make sure no messages get lost if no consumer
-        # has connected, yet.
-        if not self.bound:
-            await self.bind_queue(event_name)
+    async def declare_exchange(self, exchange_name):
+        await self.channel.exchange_declare(exchange_name=exchange_name,
+                                            type_name='topic',
+                                            durable=True)
+
 
+    async def emit(self, event_name, payload):
         # Try to json.dumps if the payload is not a string or bytes
         if not isinstance(payload, str) and not isinstance(payload, bytes):
             payload = json.dumps(payload)
-
-        await self.channel.publish(
+        domain, event_type = split_event_name(event_name)
```

Comment:
On [line 78](https://github.com/TwylaHelps/twyla.service/pull/8/files#diff-bf339141f52b6810c9ce777c339afa22R78), we are calling the variables just `domain, event` for the return values from `split_event_name` -- I think we should try to keep it consistent and more reflective of what the return values represent (especially because we have an `Event` class somewhere else that does not necessarily mean an event name).

Reply:
p.s. I vote for `event_name` since it's also what's given in the `assert` inside `split_event_name`.

Reply:
I went with calling "domain.event_type" the event name, and the second part the event_type